packages/office-ui-fabric-react/src/components/DocumentCard/DocumentCard.styles.ts

```diff
@@ -1,9 +1,9 @@
 import { getGlobalClassNames, FontSizes } from '../../Styling';
 import { IDocumentCardStyleProps, IDocumentCardStyles } from './DocumentCard.types';
-import { GlobalClassNames as previewClassNames } from './DocumentCardPreview.styles';
-import { GlobalClassNames as activityClassNames } from './DocumentCardActivity.styles';
-import { GlobalClassNames as titleClassNames } from './DocumentCardTitle.styles';
-import { GlobalClassNames as locationClassNames } from './DocumentCardLocation.styles';
+import { DocumentCardPreviewGlobalClassNames as previewClassNames } from './DocumentCardPreview.styles';
```

Comment:
Could we remove the `as` renaming here now that each export is named distinctly?

Reply:
We could... but it's much shorter this way in the rest of the file)))